src/resources/LimitRanges/LimitRangesList.jsx

```diff
@@ -54,7 +54,7 @@ export function LimitRangesList(props) {
     <ToolbarButton
       key={`create-limit-ranges`}
       data-testid={`create-limit-ranges`}
-      design="Emphasized"
+      design="Default"
```

Comment:
Desing by default is default: https://ui5.github.io/webcomponents/components/ToolbarButton/#design
Applies to all buttons